suites/quincy/rbd/tier-1_rbd.yaml

```diff
@@ -0,0 +1,181 @@
+# Tier1: RBD build evaluation
+#
+# This test suite evaluates the build to determine the execution of identified
+# regression test suites. This suite is executed against all new builds released by the
+# development team.
+#
+
```

Comment:
It would be helpful if you could add the minimal cluster conf that is required to execute this test.

Reply:
@psathyan sorry for missing, added now